src/browser/chrome.ts

```diff
@@ -157,8 +191,40 @@ export const openUrl = async ({
         msg.type() === "error" &&
         log(`JS error |`, msg.text().replace(/\n[\w\W]*/, ""), `@ [${url}]`)
     );
+
+    let counterRequestsUnfinished = 0;
+    const updateCounter = async (counter: number) => { 
+      counterRequestsUnfinished = counter;
```

Comment:
Reassign is confusing. Just pass the delta as an argument, `-1` or `1` and inc/dec